app/views/work_package_mailer/mentioned.html.erb

```diff
@@ -0,0 +1,55 @@
+<table <%= placeholder_table_styles %>>
+  <tr>
+    <%= placeholder_cell('20px', vertical: false) %>
+  </tr>
+</table>
+
+<table <%= placeholder_table_styles(width:'100%',style: 'width:100%;max-width:700px;margin-left:auto;margin-right:auto') %>>
+  <tr>
+    <td>
+      <%= render partial: 'mailer/notification_mailer_header',
+                 locals: {
+                   summary: "#{I18n.t(:'mail.work_packages.mentioned_by', user: @journal.user)}"
+                 } %>
+
+      <%= render layout: 'mailer/notification_row',
+                 locals: {
+                   work_package: @work_package,
+                   unique_reasons: [:mentioned],
+                   show_count: false
+                 } do %>
+        <table <%= placeholder_table_styles %>>
+          <tr>
+            <td style="color: #878787; line-height: 24px; font-size: 14px; white-space: normal;">
+              <%= simple_format @journal.notes %>
```

Comment:
I believe this should be 

```
format_text @journal.notes, only_path: false
```

as otherwise our styling wouldn't be applied, e.g. tables.

But of course this then poses styling problems. E.g I added an image and that then breaks out of the boundary box

![image](https://user-images.githubusercontent.com/617519/135596774-e889900e-7d1d-454d-bd0a-cc0b194a9a09.png)